src/core/symbology/qgsmarkersymbollayer.cpp

```diff
@@ -2802,40 +2802,43 @@ QgsSymbolLayer *QgsRasterMarkerSymbolLayer::create( const QVariantMap &props )
   if ( props.contains( QStringLiteral( "scale_method" ) ) )
     scaleMethod = QgsSymbolLayerUtils::decodeScaleMethod( props[QStringLiteral( "scale_method" )].toString() );
 
-  QgsRasterMarkerSymbolLayer *m = new QgsRasterMarkerSymbolLayer( path, size, angle, scaleMethod );
+  std::unique_ptr< QgsRasterMarkerSymbolLayer > m = std::make_unique< QgsRasterMarkerSymbolLayer >( path, size, angle, scaleMethod );
+  m->setCommonProperties( props );
+  return m.release();
+}
 
+void QgsRasterMarkerSymbolLayer::setCommonProperties( const QVariantMap &props )
```

Comment:
```suggestion
void QgsRasterMarkerSymbolLayer::setCommonProperties( const QVariantMap &properties )
```

I other functions (e.g. QgsRasterMarkerSymbolLayer::resolvePaths) properties is spelled in its complete form, we might as well stick to that.